purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/offlineentitlements/OfflineEntitlementsFreshInstallIntegrationTest.kt

```diff
@@ -12,7 +13,37 @@ import org.junit.runner.RunWith
 
 @Suppress("TooManyFunctions")
 @RunWith(AndroidJUnit4::class)
-class OfflineEntitlementsFreshInstallIntegrationTest : BaseOfflineEntitlementsIntegrationTest() {
+class OfflineEntitlementsFreshInstallIntegrationNoPurchasesTest : BaseOfflineEntitlementsIntegrationTest() {
```

Comment:
Note that I separated these in 2. One with initial purchases and one without. To make sure we didn't enter offline entitlements in either case without the product entitlement mapping.